test/integration/suites-windows/windows-service/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM spire-agent-windows:latest-local as spire-agent-windows
+
+FROM spire-server-windows:latest-local as spire-base
+
+COPY --from=spire-agent-windows C:/spire/bin/spire-agent.exe C:/spire/bin/spire-agent.exe
```

Comment:
why you choose to use the same container to start both services?

Reply:
So, in this initial version, I tried to reproduce a test that I ran manually on my local machine (spire server and agent on the same host). Still, I see that a more realistic scenario would be running them in different hosts, right? I can update it :)

Reply:
yeah that is 'generally' the case